sympy/vector/tests/test_coordsysrect.py

```diff
@@ -299,84 +299,74 @@ def test_evalf():
     # Variables and expressions
-    a._connect_to_standard_cartesian(((x, y, z), (x, y, z)))
-    assert a._transformation_equations() == (a.x, a.y, a.z)
+    x, y, z = symbols('x y z')
+    a = CoordSys3D('a', transformation=((x, y, z), (x, y, z)))
+    a._calculate_inv_trans_equations()
+    assert a.transformation_to_parent() == (a.x, a.y, a.z)
     assert a.lame_coefficients() == (1, 1, 1)
-    assert a._inverse_transformation_equations() == (a.x, a.y, a.z)
-    a._connect_to_standard_cartesian(((x, y, z), ((x * cos(y), x * sin(y), z))), inverse=False)
-    assert a._transformation_equations() == (a.x * cos(a.y), a.x * sin(a.y), a.z)
+    assert a.transformation_from_parent_function()(x, y, z) == (x, y, z)
+    a = CoordSys3D('a', transformation=(((x*cos(y), x*sin(y), z)), (x, y, z)))
+    assert a.transformation_to_parent() == (a.x * cos(a.y), a.x * sin(a.y), a.z)
     assert simplify(a.lame_coefficients()) == (1, sqrt(a.x**2), 1)
-    a._connect_to_standard_cartesian(((x, y, z), (x * sin(y) * cos(z), x * sin(y) * sin(z), x * cos(y))), inverse=False)
-    assert a._transformation_equations() == (a.x * sin(a.y) * cos(a.z),
-                          a.x * sin(a.y) * sin(a.z),
-                          a.x * cos(a.y))
-    assert simplify(a.lame_coefficients()) == (1, sqrt(a.x**2), sqrt(sin(a.y)**2*a.x**2))
-    # Equations
-    a._connect_to_standard_cartesian((a.x*sin(a.y)*cos(a.z), a.x*sin(a.y)*sin(a.z), a.x*cos(a.y)), inverse=False)
-    assert a._transformation_equations() == (a.x * sin(a.y) * cos(a.z),
-                          a.x * sin(a.y) * sin(a.z),
-                          a.x * cos(a.y))
-    assert simplify(a.lame_coefficients()) == (1, sqrt(a.x**2), sqrt(sin(a.y)**2*a.x**2))
-    a._connect_to_standard_cartesian((a.x, a.y, a.z))
-    assert a._transformation_equations() == (a.x, a.y, a.z)
-    assert simplify(a.lame_coefficients()) == (1, 1, 1)
-    assert a._inverse_transformation_equations() == (a.x, a.y, a.z)
-    a._connect_to_standard_cartesian((a.x * cos(a.y), a.x * sin(a.y), a.z), inverse=False)
-    assert a._transformation_equations() == (a.x * cos(a.y), a.x * sin(a.y), a.z)
-    assert simplify(a.lame_coefficients()) == (1, sqrt(a.x**2), 1)
-
-    raises(ValueError, lambda: a._connect_to_standard_cartesian((x, y, z)))
 
 
 def test_check_orthogonality():
-    a = CoordSys3D('a')
-    a._connect_to_standard_cartesian((a.x*sin(a.y)*cos(a.z), a.x*sin(a.y)*sin(a.z), a.x*cos(a.y)), inverse=False)
-    assert a._check_orthogonality() is True
-    a._connect_to_standard_cartesian((a.x * cos(a.y), a.x * sin(a.y), a.z))
-    assert a._check_orthogonality() is True
-    a._connect_to_standard_cartesian((cosh(a.x)*cos(a.y), sinh(a.x)*sin(a.y), a.z), inverse=False)
-    assert a._check_orthogonality() is True
+    x, y, z = symbols('x y z')
+    u,v = symbols('u, v')
+    a = CoordSys3D('a', transformation=((x*sin(y)*cos(z), x*sin(y)*sin(z), x*cos(y)), (x, y, z)))
```

Comment:
shouldn't the tuple `(x, y, z)` come before the expression?

That's why I suggest to simply use: `lambda x, y, z: (x*sin(y)*cos(z), x*sin(y)*sin(z), x*cos(y))`